Base/usr/share/man/man7/Tips-and-Tricks.md

```diff
@@ -0,0 +1,60 @@
+![Welcome Icon](/res/icons/32x32/app-welcome.png)
+
+## Name
+Tips and Tricks
+
+## Description
+This is a list of useful tips and tricks to help you make the most out of SerenityOS.
+
+## General
+* When on the Desktop or in File Manager, start typing and the nearest match will be selected.
```

Comment:
itʼd be good to use one of the “active” or “focus” words here for more clarity. 😉

```suggestion
* […], start typing to select the nearest match.
```

to be frank, “the nearest match” is not the best wording… any ideas to be just slightly more specific here? 🤔

Reply:
I didn't want to use the world "search" because that has wider connotations and felt I'd found a good solution.
As you type only the icons/documents in your current focus (e.g. on the desktop or in the current folder in the File Manager) are being searched through.
I think your amendment makes it read much better though, so I will change it to that!

Reply:
Or "When on the Desktop or in File Manager, start typing to select the nearest match shortcut or filename"